Gemfile

```diff
@@ -3,6 +3,10 @@ source 'https://rubygems.org'
 ruby '3.0.2'
 
 gem 'sinatra'
+gem 'puma'
+gem 'falcon'
+# gem 'webrick'
+gem 'launchy'
 
 group :test do
```

Comment:
You may also need the following gems added to your gem file.
gem 'sinatra-contrib'
gem 'thin'
gem 'webrick'